web/pgadmin/tools/sqleditor/utils/query_tool_preferences.py

```diff
@@ -208,6 +208,16 @@ def register_query_tool_preferences(self):
         )
     )
 
+    self.highlight_matched_selection = self.preference.register(
+        'Editor', 'highlight_matched_selection',
```

Comment:
again.. as per the label variable name should be `highlight_selection_matches`. Don't multiple names for same things. Keep the name consistent everywhere.